04-concept-api/references/rule.yml

```diff
@@ -109,7 +109,7 @@ methods:
       <<: *method-delete
     javascript:
       <<: *method-delete
-      method: await rule.delete();
+      method: await rule.asRemote(Transaction tx).delete();
```

Comment:
Looks like the common method needs to change, and as a result there won't be a common method any more because java needs `Transaction tx` and python needs `tx`